### DISCUSSION TITLE
Hosted Quirrel

### DISCUSSION THREAD

**Discussion by benjakugler96** [quirrel-dev/quirrel#746] [Q&A] [answered]
Hi!

I have a Blitzjs application which works perfectly in dev mode with quirrel. Now im trying to deploy it but I'm facing some problems. This is what I added so far to the .env file
QUIRREL_BASE_URL
QUIRREL_TOKEN
QUIRREL_ENCRYPTION_SECRET
SESSION_SECRET_KEY

My question is specifically about the quirrel_base_url. lets say my app is hosted in app.appname.com, then that should be the quirrel_base_url? I already tried that but it doesnt seems to work. 
Is there any change needed for the start or build script?

**Comment by Skn0tt** [comment, marked as answer]
Hi Benja! `QUIRREL_BASE_URL` should be `app.appname.com`, correct. e.g. for [quirrel.dev](https://quirrel.dev), I set it to `https://quirrel.dev`. If you leave out the `https://`, that should [be added automatically](https://github.com/quirrel-dev/quirrel/blob/930b95bc3b94cb230505c1aab3d8b19f6373ecf8/src/client/config.ts#L106).

What do you mean by "it doesn't seem to work"? What are you expecting to see, and what are you seeing? If you want to use cronjobs, mabye you're [missing a call to `quirrel ci`](https://docs.quirrel.dev/deploying) in your build script.

**Reply by benjakugler96** [reply]
@Skn0tt Thanks for the quick response!
In one of the blitz mutations I call to bookmarksProcessingQueue.enqueue({ html, userId }). This bookmarksProcessingQueue file looks like this:

```
import { Queue } from "quirrel/blitz"

export default Queue("api/nests", async (data) => {
// some code ...
}
```

But nothing inside this queue gets executed. I'm missing something?
Everything is working perfectly in development mode by running the app + quirrel. My issue is just in prod mode and I dont see anything in console. I do see some requests going to incidents

**Reply by Skn0tt** [reply]
> I do see some requests going to incidents

That's a good pointer! I had a look at your incidents (magic admin powers), and it looks like requests are continuing to be sent to either `localhost:3000` or `localhost:9181`. Can you confirm the environment variable was correctly updated? Sometimes you'll need to re-deploy.

**Reply by benjakugler96** [reply]
Right now I'm locally running the app in production mode, thats why I changed it to localhost:3000. Or locally I should always run quirrel separately? But same was happening in real prod for app.appname.com

**Reply by Skn0tt** [reply]
The Quirrel API won't be able to reach your app if it's running locally, so you'll need to test that integration in production. It's important that you yet the `QUIRREL_BASE_URL` environment variable on your production service.
